multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -504,6 +537,21 @@ func objectMapFunc(a client.Object) []reconcile.Request {
 	}
 }
 
+func endpointSliceMapFunc(a client.Object) []reconcile.Request {
+	labels := a.GetLabels()
+	svcName := labels[discovery.LabelServiceName]
+	mappedObject := types.NamespacedName{}
+	if svcName != "" {
+		mappedObject.Name = svcName
+		mappedObject.Namespace = a.GetNamespace()
+	}
+	return []reconcile.Request{
+		{
+			NamespacedName: mappedObject,
+		},
+	}
+}
+
 // serviceHandler handles Service related change.
 // ClusterIP type: update corresponding ResourceExport only when ClusterIP or Ports change.
 func (r *ServiceExportReconciler) serviceHandler(
```

Comment:
I guess rename `refreshResourceExport` to `resetResourceExport`

Reply:
Renamed